src/app/(proper_react)/(redesign)/(public)/how-it-works/components/Footer.tsx

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Props } from "../HowItWorksView";
+import { Button } from "../../../../../components/client/Button";
+import styles from "../HowItWorksView.module.scss";
+
+export const Footer = (props: Props) => {
+  const { l10n } = props;
+
+  return (
+    <footer className={styles.footer}>
+      <h2 className={styles.footerCTA}>{l10n.getString("footer-title")}</h2>
+      <div className={styles.footerCTAInput}>
+        <input
+          placeholder={l10n.getString("footer-input-placeholder")}
+          type="email"
+          name="email"
+          id="email"
```

Comment:
I _think_ this is OK, but it might have a `name`/`id` conflict with the admin:
```sh
git grep -Pn ' (id|name)="email"' src | cat

src/app/(proper_react)/(redesign)/(authenticated)/admin/dev/UserAdmin.tsx:98:              name="email"
src/app/(proper_react)/(redesign)/(authenticated)/admin/dev/UserAdmin.tsx:99:              id="email"
src/app/(proper_react)/(redesign)/(public)/how-it-works/components/Footer.tsx:19:          name="email"
src/app/(proper_react)/(redesign)/(public)/how-it-works/components/Footer.tsx:20:          id="email"
```

Reply:
Updated here:
https://github.com/mozilla/blurts-server/pull/4632/commits/9ec29911b82ddb82885897e3eb20cbdc3ba4d29f